### PR TITLE
OSDOCS#8813: Removed 1.10 references for 4.14 and above

### PR DESCRIPTION
**Pull request by snarayan-redhat**
Version(s): 4.14+

Issue: https://issues.redhat.com/browse/OSDOCS-8813

Link to docs preview:
https://68376--docspreview.netlify.app/openshift-enterprise/latest/security/cert_manager_operator/cert-manager-operator-release-notes

QE review:
- [x] QE has approved this change. @lunarwhite 
- [x] SME has approved this change. @swghosh

### CODE REVIEW

**Review by lunarwhite — COMMENTED** [review]
/lgtm

**Comment by snarayan-redhat** [comment]
@kalexand-rh this looks like a change management candidate. Can you PTAL and give an approval?

File: security/cert_manager_operator/cert-manager-operator-release-notes.adoc [hunk 1]
```diff
@@ -161,95 +161,4 @@ This is the general availability (GA) release of the {cert-manager-operator}.
...
-
-* The {cert-manager-operator} does not support using Google workload identity federation. (link:https://issues.redhat.com/browse/OCPBUGS-9998[*OCPBUGS-9998*])
-
-* When uninstalling the {cert-manager-operator}, if you select the *Delete all operand instances for this operator* checkbox in the {product-title} web console, the Operator is not uninstalled properly. As a workaround, do not select this checkbox when uninstalling the {cert-manager-operator}. (link:https://issues.redhat.com/browse/OCPBUGS-9960[*OCPBUGS-9960*])
+* After uninstalling the {cert-manager-operator} by using the web console, the {cert-manager-operator} does not remove the cert-manager controller, CA injector, and Webhook pods automatically from the `cert-manager` namespace. As a workaround, you can manually delete the cert-manager controller, CA injector, and Webhook pod deployments present in the `cert-manager` namespace. (link:https://issues.redhat.com/browse/OCPBUGS-13679[*OCPBUGS-13679*])
```
Comment (dfitzmau) [inline security/cert_manager_operator/cert-manager-operator-release-notes.adoc:164]: Should the Jira have the release notes field populated (such as Release Note Type = Known issue)?

Comment (snarayan-redhat) [reply security/cert_manager_operator/cert-manager-operator-release-notes.adoc:164]: @swghosh did an engg bug fix this compatibility? If yes, we would need to incorporate the above comment by @dfitzmau

Comment (snarayan-redhat) [reply security/cert_manager_operator/cert-manager-operator-release-notes.adoc:164]: @dfitzmau regarding your second comment above, it is an existing content that you would see in line 164. Strangely, github has detected changes. All this PR is does is removal of text.

Comment (dfitzmau) [reply security/cert_manager_operator/cert-manager-operator-release-notes.adoc:164]: Yeah, typically the following release note fields on a Jira would be populated:

* Release Note Text: Can use note from this PR. 
* Release Note Type: Known issue
* Release Note Status: `In progress` is probably the best status. Set `Done` when the known issue is fixed.

Comment (dfitzmau) [reply security/cert_manager_operator/cert-manager-operator-release-notes.adoc:164]: Ah, OK. Please ignore my second point. Shubha. I can see that the primary target content has been removed.

**Comment by dfitzmau** [comment]
#
/remove-label peer-review-in-progress

**Pull request merged**